docs/pom.xml

```diff
@@ -19,6 +19,7 @@
     <docs.main>spring-cloud-azure</docs.main>
     <main.basedir>${basedir}/..</main.basedir>
     <docs.whitelisted.branches>${project.version}</docs.whitelisted.branches>
+    <docs.beta.release.branches>feature/spring-cloud-azure-passwordless-connection</docs.beta.release.branches>
```

Comment:
Why create a new property? 
1. `docs.whitelisted.branches` can be modified to the feature name.
2. `docs.whitelisted.branches` support a list. We can do like this: `<docs.whitelisted.branches>${project.version}, feature/spring-cloud-azure-passwordless-connection</docs.whitelisted.branches>`